delta-proto/src/main/java/io/cdap/delta/proto/TableTransformation.java

```diff
@@ -49,7 +49,9 @@ public String getTableName() {
    */
   public void validate() {
     if (tableName == null || tableName.isEmpty()) {
-      throw new IllegalArgumentException("Table name of a TableTransformation should not be null or empty");
+      throw new IllegalArgumentException(String.format("Table name of a TableTransformation that contains below " +
+                                                         "ColumnTransformations : %s is null or empty.",
+                                                       columnTransformations));
```

Comment:
`columnTransformations` can be `null` so would be good to check that here.

Reply:
hmm... don't quite understand, if it's null, what should we print in the string ?

Reply:
discussed offline, will remove the columnTransformations in the message here.